Sources/ClientRuntime/Networking/Http/HttpContext.swift

```diff
@@ -9,61 +9,73 @@ public class HttpContext: MiddlewareContext {
     public init(attributes: Attributes) {
         self.attributes = attributes
     }
-    // FIXME: Move all defined keys to separate file as constants to be used elsewhere
-    public func getPath() -> String {
-        return attributes.get(key: AttributeKey<String>(name: "Path"))!
-    }
 
-    public func getMethod() -> HttpMethodType {
-        return attributes.get(key: AttributeKey<HttpMethodType>(name: "Method"))!
-    }
+//    public func getAuthSchemes() -> Attributes {
+//        return attributes.get(key: AttributeKeys.authSchemes)!
+//    }
```

Comment:
Should this remain commented out?

Reply:
Yes, it's commented temporary - will be uncommented in next milestone PR (auth scheme changes PR)

Reply:
Removed from PR